docs/user-docs/latest/users-roles-permissions/managing-administrators.md

```diff
@@ -0,0 +1,103 @@
+# Managing administrator accounts
+
+Administrators are the users of an admin panel of a Strapi application. Administrator accounts and roles are managed with the Role-Based Access Control (RBAC) feature. It is available in the *Administration panel* section of the Settings interface, accessible from *General > Settings* in the main navigation of the admin panel.
+
+The *Administration panel* section is divided into 2 sub-sections: *Roles* (see [Configuring administrator roles](configuring-administrator-roles.md)) and *Users*.
+
+![Edit an administrator account](../assets/users-permissions/administrator_list-view.png)
+
+The *Users* sub-section of *Administration panel* displays a table listing all the administrators of your Strapi application.
+
+From this interface, it is possible to:
+
+- make a textual search (1) to find specific administrators,
+- sort the administrator accounts (2) to make it easier to view the list of administrator accounts,
+- set filters (2) to find specific administrators,
+- create a new administrator account (see [Creating a new account](#creating-a-new-account)),
+- delete an administrator account (see [Deleting an account](#deleting-an-account)),
+- or access information regarding an administrator account, and edit it (see [Editing an account](#editing-an-account)).
+
+For each administrator listed in the table, their main account information are displayed, including name, email and attributed role. The status of their account is also indicated: active or inactive, depending on whether the administrator has already logged in to activate the account or not.
+
+<!---
+### Filtering accounts
```

Comment:
Whats there a reason the filtering was commented out for now? Generally best to add in a comment as to why the content is being withheld (my memory can be terrible sometimes, or if someone else steps in to uncomment it)

Reply:
Oh thank you for reminding me of this one, I forgot to delete it! We don't need it in the documentation I think.